### PR TITLE
[pr deploy] Add cloud build

### PR DESCRIPTION
**Pull request by estherkim**
Now you can trigger a deployment by pushing a tag described in the DEPLOYMENT doc

### CODE REVIEW

File: pr-deploy/app.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+runtime: nodejs
+env: flexible
```
Comment (rsimha) [inline pr-deploy/app.yaml:2]: I've seen this setting in other files. Needed here?
```yaml
beta_settings:
  cloud_sql_instances: amp-pr-deploy-bot:us-central1:amp-pr-deploy-bot
```

**Review by rsimha — APPROVED** [review]
Thanks for the quick fix!

**Review by rsimha — COMMENTED** [review]
IIRC, there needs to be a `deploy-tag` npm script in `pr-deploy/package.json`, right?

File: pr-deploy/app.yaml (hunk 1 above)
Comment (estherkim) [reply pr-deploy/app.yaml:2]: Nope, pr deploy doesn't use Cloud SQL

**Comment by estherkim** [comment]
I haven't bothered to add it because it doesn't work on macOS and it's a simple git command.

**Comment by rsimha** [comment]
> I haven't bothered to add it because it doesn't work on macOS and it's a simple git command.

Gotcha. The command was fixed for macOS in #1144. Agree, it isn't critical to add.

**Pull request merged**